public/gameInfo.php

```diff
@@ -1179,8 +1181,30 @@ function resize() {
     <?php view()->share('sidebar', true) ?>
     <aside>
         <?php
+        $boxArtImgSrc = media_asset($gameData['ImageBoxArt']);
+
+        $boxArtImgSize = Cache::remember(
+            'game:box-art-dimensions:' . $gameID,
+            Carbon::now()->addDays(7),
+            function () use ($boxArtImgSrc) {
+                $boxArtImgSize = @getimagesize($boxArtImgSrc);
```

Comment:
While this is probably good for production, it's very impactful on my dev site. Hitting random pages takes (on average) 1.2 seconds longer than on master - presumably because it has to download the asset from the real server to calculate the size.
```
MEDIA_URL=https://media.retroachievements.org
```

I'm guessing even on production, `$boxArtImgSrc` will be a URL so it'll be more complex than just reading a file from the host machine.

Reply:
the new media system will include metadata like dimensions in a model which will make it easier to set sizes explicitly. disk i/o is very expensive.
i'd favor waiting a bit longer to address the layout shift until we have the data in place?

Reply:
Not a problem - though I do wonder if we could start storing this dimension data somehow now in the DB for these media assets which are causing the layout shifts. We could capture the data on upload for new assets and find some clever way to backfill data for existing assets.

Reply:
the media management will do just that. see `media` table.
all media that currently exists will be imported there, i'd like to adjust a few other things first though as it impacts all areas of the service.

Reply:
This makes sense to me 👍 I will close this PR.